cfgov/v1/models/base.py

```diff
@@ -109,6 +109,10 @@ class CFGOVPage(Page):
     # Panels
     sidefoot_panels = [
         StreamFieldPanel('sidefoot'),
+        InlinePanel(
+            'related_posts_categories',
+            label="Related Posts Categories"
+        )
```

Comment:
The placement of this is not ideal, however there is no real alternative since putting it inside the Related Posts module itself is not possible